pkg/cluster/delete.go

```diff
@@ -303,7 +303,7 @@ func (m *manager) Delete(ctx context.Context) error {
 		return err
 	}
 
-	if !m.env.IsDevelopmentMode() {
```

Comment:
maybe worth renaming IsDevelopmentMode to something like "is on your laptop or in CI mode"?